src/app/screens/Dashboard/screens/Detail/layout.tsx

```diff
@@ -47,7 +36,7 @@ function DetailContainer({ title, componentCode, onDownload }: Props) {
         </a>
       </div>
       <div className={styles.detailsComponent}>
-        <div ref={shadowElem} />
+        <RecipePreview component={component} />
```

Comment:
Shouldn't the name be `recipe` too? 🤔

Reply:
I renamed component to recipe in all cases, I hope that's what you meant 😅